src/controllers/user/dtos/request/user-credentials.dto.ts

```diff
@@ -3,7 +3,7 @@ import { IsNotEmpty, IsString } from 'class-validator';
 export class UserCredentialsDto {
   @IsString()
   @IsNotEmpty()
-  username: string;
+  email: string;
```

Comment:
Validaria que sea un email en vez de solo string
```suggestion
  @IsEmail()
  @IsNotEmpty()
  email: string;
```